src/com/facebook/buck/cli/PublishCommand.java

```diff
@@ -75,6 +75,19 @@
       usage = "Just print the artifacts to be published")
   private boolean dryRun = false;
 
+  @Option(
+      name = "--username",
+      aliases = "-u",
+      usage = "User name to use to authenticate with the server")
+  private String username;
+
+  @Option(
+      name = "--password",
+      aliases = "-p",
+      usage = "Password to use to authenticate with the server")
```

Comment:
Perhaps we copy what the AWS stuff does? It has a multi-layered approach, starting with a file in the homedir, moving to ENV variables, and then finally using values specified on the command line. We could even do he whole `AWS_PROFILE` jazz for the homedir stuff.